integration/src/main/java/scripts/testscripts/CloneWorkspace.java

```diff
@@ -269,7 +269,7 @@ protected void doUserJourney(
     assertNotNull(destinationWorkspaceId, "Destination workspace ID available immediately.");
     assertEquals(destinationUserFacingId, cloneResult.getWorkspace().getDestinationUserFacingId());
     final WorkspaceDescription destinationWorkspaceDescription =
-        cloningUserWorkspaceApi.getWorkspace(destinationWorkspaceId);
+        cloningUserWorkspaceApi.getWorkspace(destinationWorkspaceId, /*minimumHighestRole=*/ null);
```

Comment:
An alternative would be to have `getWorkspace` do what it did and add a `getWorkspaceFiltered` that would take the extra arg. I would guess that 95% of our getWorkspace usage is with the current semantics.

Reply:
We have to pass in `minimumHighestRole` because this is the API as defined by YAML. This PR added `minimumHighestRole` as an optional query parameter.

Unfortunately, the way we generate our Java client, you pass into something for each query parameter, even if you don't want to set that parameter. I explored this and found how to generate Java clients without this behavior. Basically, you want to get your mustache from https://github.com/OpenAPITools/openapi-generator and not https://github.com/swagger-api/swagger-codegen-generators, and use `x-group-parameters` and/or `useSingleRequestParameter`.